cli/ops/dispatcher.rs

```diff
@@ -0,0 +1,69 @@
+use crate::state::ThreadSafeState;
+use deno::CoreOp;
+use deno::OpId;
+use deno::PinnedBuf;
+use std::collections::BTreeMap;
+use std::sync::atomic::{AtomicU32, Ordering};
+use std::sync::RwLock;
+
+#[derive(Default)]
+pub struct OpDispatcher<H>
+where
+  H: Copy + 'static,
+{
+  pub op_registry: RwLock<BTreeMap<OpId, H>>,
+  pub name_registry: RwLock<BTreeMap<String, OpId>>,
```

Comment:
Since `op_id` is effectively an offset we could try with `Vec` instead of `BTreeMap`